src/main/java/app/model/Map.java

```diff
@@ -127,6 +128,29 @@ public void addFurniture(SettingsObject obj)
         }
     }
 
+    public MemoryGraph createFullGraph()
+    {
+        MemoryGraph<GraphCell, DefaultWeightedEdge> fullGraph = new MemoryGraph<>(20);
+        for (int i=10; i<height; i+=20)
+        {
+            for(int j = 10; j < width; j += 20)
```

Comment:
I know we currently always set moveLenght to 20, but shouldn't it be calling that variable (or walkLength) from settings?